resources/ibm-credentials.env

```diff
@@ -9,4 +9,13 @@ SERVICE_1_APIKEY=V4HXmoUtMjohnsnow=KotN
 SERVICE_1_CLIENT_ID=somefake========id
 SERVICE_1_CLIENT_SECRET===my-client-secret==
 SERVICE_1_AUTH_URL=https://iamhost/iam/api=
-SERVICE_1_URL=service1.com/api
\ No newline at end of file
+SERVICE_1_URL=service1.com/api
+
+# Service2 configured with IAM w/scope
+SERVICE_2_AUTH_TYPE=iam
+SERVICE_2_APIKEY=V4HXmoUtMjohnsnow=KotN
```

Comment:
No matter what @dpopp07 says, I'm ok with you spelling this "john" instead of "jon" :)